<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frm_User_ChangePwd</class>
 <widget class="QDialog" name="Frm_User_ChangePwd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbl_pwd0">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>原密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_pwd1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>新密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_pwd2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_pwd0">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_pwd1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_pwd2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="button_yes">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_no">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
